src/client/Components/Calendar.jsx

```diff
@@ -0,0 +1,20 @@
+export default function Results() {
```

Comment:
Keep the file names the same as the component name to make it easier to navigate the project, this function should probably be called `Calendar`